### PR TITLE
Hide/Unhide Table of Content in Quality Control Page

### PR DESCRIPTION
**Pull request by subinasr**

### CODE REVIEW

File: src/views/Entries/QualityControl/index.tsx [hunk 1]
```diff
@@ -380,55 +397,58 @@ function QualityControl(props: Props) {
                             verifiedCountSelector={verifiedCountSelector}
                             unverifiedCountSelector={unverifiedCountSelector}
                             onChange={handleSelection}
-                            searchValue={searchValue}
```
Comment (AdityaKhatri) [inline src/views/Entries/QualityControl/index.tsx:383]: add this back

File: src/views/Entries/QualityControl/index.tsx [hunk 2]
```diff
@@ -380,55 +397,58 @@ function QualityControl(props: Props) {
...
                             value={tocFilters}
-                            defaultCollapseLevel={defaultCollapseLevel}
+                            defaultCollapseLevel={5}
                             multiple
                         />
```
Comment (AdityaKhatri) [inline src/views/Entries/QualityControl/index.tsx]: add this back

File: src/views/Entries/QualityControl/index.tsx [hunk 3]
```diff
@@ -337,39 +347,46 @@ function QualityControl(props: Props) {
...
-                                    className={styles.toggleExpandButton}
-                                    onClick={handleToggleExpand}
-                                    transparent
-                                    iconName={defaultCollapseLevel === collapseLevel.min ? 'chevronDown' : 'chevronUp'}
-                                    title={defaultCollapseLevel === collapseLevel.min ?
-                                        _ts('entries.qualityControl', 'expandAll')
-                                        : _ts('entries.qualityControl', 'collapseAll')
-                                    }
+            <div className={styles.wrappingContainer}>
+                {isExpanded ?(
```
Comment (AdityaKhatri) [inline src/views/Entries/QualityControl/index.tsx]:
```suggestion
                {isExpanded ? (
```

File: src/views/Entries/QualityControl/index.tsx [hunk 4]
```diff
@@ -337,39 +347,46 @@ function QualityControl(props: Props) {
...
-                                placeholder={_ts('entries.qualityControl', 'searchInputPlacehoder')}
-                                keySelector={idSelector}
-                                labelSelector={labelSelector}
-                                showLabel
-                                showHintAndError={false}
-                            />
-                        </header>
+                            </header>
+                        </div>
+                        { pending && <LoadingAnimation /> }
```
Comment (AdityaKhatri) [inline src/views/Entries/QualityControl/index.tsx]: Move LoadingAnimation under container called `right`

File: src/views/Entries/QualityControl/styles.scss [hunk 5]
```diff
@@ -51,18 +51,29 @@
...
+        }
+
+        .expand-button {
+            width: 25px;
+            height: 25px;
         }
 
         .right {
             display: flex;
             flex-direction: column;
```
Comment (AdityaKhatri) [inline src/views/Entries/QualityControl/styles.scss:84]: Add position relative to fix LoadingAnimation

File: src/views/Entries/QualityControl/index.tsx [hunk 6]
```diff
@@ -386,49 +402,53 @@ function QualityControl(props: Props) {
                             multiple
                         />
                     </div>
+                ) :(
+                    <Button
```
Comment (subinasr) [inline src/views/Entries/QualityControl/index.tsx]: This button is still not an overflow button. I need help on this.

**Pull request merged**